compiler/src/dotty/tools/backend/jvm/BCodeIdiomatic.scala

```diff
@@ -29,6 +29,7 @@ trait BCodeIdiomatic {
     case "jvm-1.6"     => asm.Opcodes.V1_6
     case "jvm-1.7"     => asm.Opcodes.V1_7
     case "jvm-1.8"     => asm.Opcodes.V1_8
+    case "jvm-9"       => asm.Opcodes.V9
```

Comment:
It looks like we don't have testcases which are run with `-target jvm-9` currently, that would be very useful to validate that the changes in this PR indeed work correclty on Java 9 (since these tests can't be run on Java 8, they'll need to be excluded by default and only enabled in the CI when we run Java 11)